package.yaml

```diff
@@ -1,5 +1,5 @@
 name:                polysemy-zoo
-version:             0.6.0.1
+version:             0.7.0.0
```

Comment:
Please leave the version as it was, I'll get confused the next time I have to release it otherwise! But thanks for the thought!